test/testlib/nuodb_admin_utilities.go

```diff
@@ -130,4 +131,33 @@ func StartAdmin(t *testing.T, options *helm.Options, replicaCount int, namespace
 			helm.Install(t, options, "nuodb/admin ", helmChartReleaseName)
 		}
 	})
-}
\ No newline at end of file
+}
+
+func GetLoadBalancerPolicies(t *testing.T, namespaceName string, adminPod string) (map[string]NuoDBLoadBalancerPolicy, error) {
```

Comment:
so far we have followed the `TerraTest` naming convention of appending an `E` to the end of a getter function that can return an error. Functions without the `E` assert internally. Hence this function should be called `GetLoadBalancerPoliciesE`

applies to the ones below too